packages/orchestration/test/examples/swapExample.test.ts

```diff
@@ -21,12 +24,21 @@ test('start', async t => {
 
   const { publicFacet } = await E(zoe).startInstance(
     installation,
-    {},
+    { USDC: usdc.issuer },
     {},
     privateArgs,
   );
 
   const inv = E(publicFacet).makeSwapAndStakeInvitation();
 
   t.is((await readInvitation(inv)).description, 'Swap for TIA and stake');
+
+  const ten = usdc.units(10);
+  const userSeat = await E(zoe).offer(
+    inv,
+    harden({ give: { USDC: ten } }),
+    harden({ USDC: usdc.mint.mintPayment(ten) }),
```

Comment:
While `harden()` here is correct per Jessie rules, we tend to rely on the practical expedient that all args to an `E` method call get hardened. I wonder... did that not work here somehow?

Reply:
not sure how those got there

Reply:
hmm, I think it was my AI coding assistant's autocomplete